indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/helpers/WalletHelper.kt

```diff
@@ -0,0 +1,33 @@
+package com.luxoft.blockchainlab.hyperledger.indy.helpers
+
+import com.luxoft.blockchainlab.hyperledger.indy.models.WalletConfig
+import com.luxoft.blockchainlab.hyperledger.indy.models.WalletPassword
+import com.luxoft.blockchainlab.hyperledger.indy.utils.SerializationUtils
+import com.luxoft.blockchainlab.hyperledger.indy.utils.getRootCause
+import org.hyperledger.indy.sdk.wallet.Wallet
+import org.hyperledger.indy.sdk.wallet.WalletExistsException
+
+/**
+ * Helps to manage wallets
+ */
+object WalletHelper {
+    /**
+     * Creates (or not if exists) all needed wallet files and returns [Wallet]
+     *
+     * @param config: [WalletConfig] - wallet configuration
+     * @param password: [WalletPassword] - wallet credentials
+     * @return: [Wallet] - target wallet
+     */
+    fun getWallet(config: WalletConfig, password: WalletPassword): Wallet {
```

Comment:
What is WalletConfig and WalletPassword object move here as well? To keep them close to each other

Reply:
Ok.